docs/examples/te_llama/te_llama.py

```diff
@@ -142,32 +140,38 @@ def replace_params(hf_state_dict, te_state_dict):
         m = re.match(layer_prefix_pat, param_key)
         if m is not None:
             all_layer_prefixes.add(m.group())
+    
+    GATE_PROJ_SIZE = 11008
```

Comment:
Can we use `config` object to get this value? We could pass `config` object to `replace_params` method.